post.md

```diff
@@ -221,7 +221,7 @@ To belabour the point: there's three outcomes of that call.
 
 ## How do we... uh... reanimate?
 
-How do we let react know when the promise has fulfilled so it can have another try?
+How do we let React know when the promise has fulfilled so it can have another try?
```

Comment:
another try at what? Are we trying again after a fail or going back into loading because there's more data / data has changed? 

It was somewhere around here I started getting twitchy about all these promises getting created for the memory side, which you go into later.